### DISCUSSION TITLE
Capacitor - customize environments

### DISCUSSION THREAD

**Discussion by developerSave** [quasarframework/quasar#14954] [CLI - Capacitor / Cordova modes] [answered]
Hello everyone,
I have a big problem:
I don't know how to customize environments in my Quasar-Capacitor Android project and I need to create 3 environments: Dev, Beta and Prod.
I created .env-prod, .env-beta and .env-dev files, but I don't know where I can change the scripts for them.

Which configs files I need to modify? Is there a guide for that?

Thank you,
Lorenzo

**Comment by MilosPaunovic** [comment, marked as answer]
What exactly needs to differ for those environments?

First step would be to take a look at this
https://quasar.dev/quasar-cli-webpack/handling-process-env

**Reply by developerSave** [reply]
Hi Milos,
I noticed that I was into a web-app kind of mindset while for a mobile app it's different and I was thinking that I needed 3 environments for it to work properly.
I just need to understand how to let my apk point my production url (same url of my backend in java, that my cors accept).

Thank you!

**Reply by metalsadman** [reply]
either set your server for cors, or use https://capacitorjs.com/docs/apis/http

**Reply by metalsadman** [reply]
as for env variables https://quasar.dev/quasar-cli-webpack/handling-process-env#using-dotenv, also join our discord and search for it, many discussions about it that you can learn from. https://chat.quasar.dev/

**Reply by developerSave** [reply]
thank you!
